podman_compose.py

```diff
@@ -1380,8 +1378,7 @@ def compose_build_parse(parser):
         help="attempt to pull a newer version of the image", action='store_true')
     parser.add_argument("--pull-always",
         help="attempt to pull a newer version of the image, Raise an error even if the image is present locally.", action='store_true')
-    parser.add_argument("--build-arg",metavar="KEY=VAL",
-                            help="Set build-time variables for services.", action='append')
+
```

Comment:
I think it would be really nice if you would just drop this and the adding commit.

Reply:
I don't understand. Can you explain please ?

Reply:
You rebase with git and just drop the original commit which added those lines in https://github.com/containers/podman-compose/pull/142/commits/e5c04a607cb5c5f83073fe5ff9a1dddfaba8da2e and the one removing it.

Reply:
ok thanks, I did it.

Reply:
I have no idea what you did but you somehow amended the commits all again instead of overwriting them.

Reply:
Yes something goes wrong. I fix it. The build arg commit is deleted